test/integration/testsuite_test.go

```diff
@@ -0,0 +1,92 @@
+package test_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+type VersionAnswer struct {
+	Version          string `json:"version"`
+	Commit           string `json:"commit"`
+	OpenshiftVersion string `json:"openshiftVersion"`
+	Embedded         bool   `json:"embedded"`
+}
+
+type StatusAnswer struct {
+	CRCStatus        string `json:"crcStatus"`
+	OpenshiftStatus  string `json:"openshiftStatus"`
+	OpenshiftVersion string `json:"openshiftVersion"`
+	DiskUsage        int    `json:"diskUsage"`
+	DiskSize         int    `json:"diskSize"`
+	CacheUsage       int    `json:"cacheUsage"`
+	CacheDir         string `json:"cacheDir"`
+}
+
+var credPath string
+var userHome string
+var versionInfo VersionAnswer
+
+var bundleLocation string
+var pullSecretLocation string
+
+func TestTest(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+
+	junitfile := fmt.Sprintf("integration_%s.xml", time.Now().Format("2006-01-02 3:4:5"))
+	junitReporter := reporters.NewJUnitReporter(filepath.Join("out", junitfile))
+	RunSpecsWithDefaultAndCustomReporters(t, "Test Suite", []Reporter{junitReporter})
+	// RunSpecs(t, "Test Suite")
+}
+
+var _ = BeforeSuite(func() {
+
+	// set userHome
+	usr, err := user.Current()
+	if err != nil {
+		Expect(err).NotTo(HaveOccurred())
+	}
+	userHome = usr.HomeDir
+
+	// set credPath
+	credPath = filepath.Join(userHome, ".crc", "machines", "crc", "id_rsa")
+
+	// find out if bundle embedded in the binary
+	raw := RunCRCExpectSuccess("version", "-o", "json")
+	err = json.Unmarshal([]byte(raw), &versionInfo)
+
+	Expect(err).NotTo(HaveOccurred())
+
+	// bundle location
+	if !versionInfo.Embedded {
+		bundleLocation = os.Getenv("BUNDLE") // this env var should contain location of bundle
```

Comment:
The env variable name should be `BUNDLE_PATH` or `BUNDLE_LOCATION`

Reply:
I want it to be different than `BUNDLE_LOCATION` used with e2e tests. Went for `BUNDLE_PATH`.